src/games/kalambury/board/GameBoard.js

```diff
@@ -21,14 +35,28 @@ const GameBoard = ({
   canChangePhrase,
   ChangePhrase,
   t,
-}) => (
+}) => {
+
+  const [lastSuccess, setLastSuccess] = useState(false);
+  const previousUserGuesses = getUserActions(actions, playerID, "guess")
+
+  useEffect(() => {
+    let guess = previousUserGuesses[0];
+    if (!guess) {
+      return;
+    }
+    setLastSuccess(guess.success);
+    return () => setLastSuccess(false); // Do cleanup of state 
+  });
```

Comment:
If you don't specify any effect dependency it will be executed even for changes that aren't relevant.

Reply:
But even after specifying `actions` or `previousUserGuesses` as a dependency this gets executed constantly without any user interaction. Why does it happen when `actions` don't change? @pastDexter

Reply:
What do you mean?

Reply:
My reference https://reactjs.org/docs/hooks-effect.html#example-using-hooks-1

Reply:
You can pass a second argument to `useEffect` which is an array of properties that must change in order to execute this effect: https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects
But it seems like it doesn't make any difference here, because `actions` are being updated constantly.
Try adding something like this and see console log:
```
  useEffect(() => {
    console.log("New actions", actions);
  }, [actions]);
```

Reply:
The is flooded by the message :D

Reply:
I've tried using `actions.length` but the value is constant 👀

Reply:
Fixed 🚀

Reply:
Not all the time. If you have one wrong answer and send a good answer then, and you're the next one drawing then it doesn't trigger, because you had 2 actions before starting to draw and you still have 2 after starting 🙈
It seems like we don't actually need a guesses collection here, only the latest one. We could maybe trigger on `id` change

Reply:
But I'm using `previousUserGuesses.length` which is changing with every guess if I'm not mistaken @Bombasarkadian